### PR TITLE
Documentation updates based on onboarding experience

### PR DESCRIPTION
**Pull request by esuomi**
A few minor things I noticed while trying to get uttu (and enki) to run properly, plus a general formatting pass of the whole README to make it consistent with added hints in code blocks to allow for viewers to render the snippets with syntax highlighting.

fixes https://github.com/entur/uttu/issues/316
closes https://github.com/entur/uttu/pull/319

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,84 +74,104 @@ If you don't use Google PubSub, sett this property:
...
+
 #### Via Docker
 
 Install Docker.
 
-```bash
-docker run --name=uttu -d -e POSTGRES_USER=uttu -e POSTGRES_PASSWORD=uttu -e POSTGRES_DB=uttu -p 5432:5432 -v db_local:/var/lib/postgresql --restart=always postgis/postgis:13-3.3
+```shell
+docker run --platform linux/amd64 --name=uttu -d -e POSTGRES_USER=uttu -e POSTGRES_PASSWORD=uttu -e POSTGRES_DB=uttu -p 
+5432:5432 -v db_local:/var/lib/postgresql --restart=always postgis/postgis:13-3.3
```
Comment (testower) [inline README.md:90]: For easier copy/paste directly into terminal:

```shell
docker run \
    --platform linux/amd64 \
    --name=uttu \
    -d \
    -e POSTGRES_USER=uttu \
    -e POSTGRES_PASSWORD=uttu \
    -e POSTGRES_DB=uttu \
    -p 5432:5432 \
    -v db_local:/var/lib/postgresql \
    --restart=always \
    postgis/postgis:13-3.3
```

File: README.md [hunk 2]
```diff
@@ -65,84 +74,104 @@ If you don't use Google PubSub, sett this property:
...
 Now a Docker container is running in the background. Check its status with `docker ps`.
 
-To stop, find its ID from `docker ps`, and run `docker stop theid` (beginning of hash). To restart it, find the ID from `docker container list` and run `docker restart theid`.
+To stop, find its ID from `docker ps`, and run `docker stop theid` (beginning of hash). To restart it, find the ID from 
+`docker container list` and run `docker restart theid`.
+
+Run the [database initialization script](./src/main/resources/db_init.sh).
 
-Run the [script](./src/main/resources/db_init.sh).
+Run the [database init script](./src/main/resources/db_init.sh).
```
Comment (testower) [inline README.md:100]: Seems to be repetition if L98, so remove either this or that.

File: README.md [hunk 3]
```diff
@@ -65,84 +74,104 @@ If you don't use Google PubSub, sett this property:
...
+/services/flexible-lines/providers/graphql
+
+Provider-specific GraphQL endpoint (replace {providerCode} with provider's codespace code):
+
+    /services/flexible-lines/{providerCode}/graphql
 
 ## Netex Export
-This api exports generated netex file to gcp storage, which is used to build graph.
+
+Uttu provides possibility (via provider specific GraphQL API) to export generated NeTEx file to a blobstore repository.
```
Comment (testower) [inline README.md:126]: "Provides possibility" is a bit misleading. The blobstore bean is not optional:

Uttu exports (via provider specific GraphQL API) generated NeTEx file to a blobstore repository.

File: README.md (hunk 2 above)
Comment (esuomi) [reply README.md]: This was an accidental rebase artifact. Fixed.

**Pull request merged**